app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala

```diff
@@ -36,38 +37,29 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-object DesConnector extends DesConnector with ServicesConfig {
-  override val serviceUrl: String = baseUrl("des-hod")
-  override val authToken: String = s"Bearer ${getConfString("des-hod.authorizationToken", "")}"
-  override val environment: String = getConfString("des-hod.environment", "")
+class DesConnector @Inject()(environment: Environment,
+                             configuration: Configuration,
+                             desSummaryRepository: DesSummaryRepository,
+                             desNIRecordRepository: DesNIRecordRepository,
+                             desLiabilitiesRepository: DesLiabilitiesRepository,
+                             metrics: MetricsService) extends ServicesConfig {
 
+  val serviceUrl: String = baseUrl("des-hod")
+  val authToken: String = s"Bearer ${getConfString("des-hod.authorizationToken", "")}"
```

Comment:
You could get this from the Configuration

Reply:
Done